.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -671,6 +676,36 @@
         <ItemGroup>
             <FileWrites Include="$(_MauiManifestStampFile)" />
         </ItemGroup>
+
+    </Target>
+
+    <Target Name="_MauiManifestMergerAfterTarget"
+            AfterTargets="_ManifestMerger"
+            Condition=" '$(_ResizetizerIsAndroidApp)' == 'True' And '$(AndroidManifestMerger)' == 'manifestmerger.jar' "
+            DependsOnTargets="_MauiGenerateAndroidManifestAttributes" />
+
+    <Target Name="_MauiGenerateJavaStubsAfterTarget"
+            AfterTargets="_GenerateJavaStubs"
+            Condition=" '$(_ResizetizerIsAndroidApp)' == 'True' And '$(AndroidManifestMerger)' != 'manifestmerger.jar' "
+            DependsOnTargets="_MauiGenerateAndroidManifestAttributes" />
+
+    <Target Name="_MauiGenerateAndroidManifestAttributes"
+            Inputs="$(MSBuildThisFileFullPath);$(_ResizetizerTaskAssemblyName);$(_ResizetizerInputsFile);$(_MauiSplashInputsFile);$(IntermediateOutputPath)android\AndroidManifest.xml"
+            Outputs="$(_MauiManifestStampFile);$(IntermediateOutputPath)android\AndroidManifest.xml"
+            Condition=" '$(_ResizetizerIsAndroidApp)' == 'True' ">
+
+        <GenerateAndroidManifestXml
```

Comment:
Adding the generation of AndroidManifest.xml to Resizetizer. Currently Android, Windows and Tizen update the manifests. iOS and Mac Catalyst is coming soon.